ksmt-core/src/main/kotlin/org/ksmt/solver/util/KExprLongConverterBase.kt

```diff
@@ -0,0 +1,137 @@
+package org.ksmt.solver.util
+
+import it.unimi.dsi.fastutil.longs.LongArrayList
+import org.ksmt.expr.KExpr
+import org.ksmt.solver.util.KExprConverterBase.Companion.ExprConversionResult
+import org.ksmt.solver.util.KExprConverterBase.Companion.argumentsConversionRequired
+import org.ksmt.solver.util.KExprConverterBase.Companion.checkArgumentsSizeMatchExpected
+import org.ksmt.sort.KSort
+
+/**
+ * Specialized version of [KExprConverterBase] for Long native expressions.
+ * */
```

Comment:
Something wrong with this comment and added classes names. We have three classes, two for longs and one for int (why a different number tho?), both classes for long have the same comment about `KExprConverterBase` while one of them is about internalizing. Please, make it more obvious, which classes are specialized by these and why it is required two classes for longs, but only one for int

Reply:
We don't have specialized version of `KExprConverter` for int because we don't need it yet.